BepInEx/Configuration/TypeConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace BepInEx.Configuration
+{
+	/// <summary>
+	/// A serializer/deserializer combo for some type(s). Used by the config system.
+	/// </summary>
+	public class TypeConverter
```

Comment:
I've talked with horse about using a type that's already in .NET (System.ComponentModel.TypeConverter), however I'm not sure if it's possible to be used like this
https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.typeconverter?view=netframework-4.8

Interested to hear your thoughts on the situation

Reply:
Dropped a word on discord.